Makefile

```diff
@@ -60,7 +60,7 @@ ifeq ($(RUN_CI), 1)
 	RELEASE_VERSION := None
 endif
 
-LDFLAGS += -X "$(PD_PKG)/pkg/versioninfo.PDReleaseVersion=$(RELEASE_VERSION)"
+LDFLAGS = -X "$(PD_PKG)/pkg/versioninfo.PDReleaseVersion=$(RELEASE_VERSION)"
```

Comment:
The local LDFLAGS variable is inherited here, and since I don't know where this variable is set locally, an error is reported here.

```shell
# In my local env
echo $LDFLAGS  
-L/opt/homebrew/opt/protobuf@3/lib
```

![img_v3_02d5_bb52df28-2704-43a7-915f-90a93dedf6bg](https://github.com/user-attachments/assets/0689f280-36ff-4df7-a346-8e5bf90ad75b)

So, I think it does not need inherited here

Reply:
We should inherit LDFLAGS and keep it the same as the other components.

Reply:
ok, I reverted it